system_tests/pubsub.py

```diff
@@ -135,7 +135,6 @@ def test_create_subscription_w_ack_deadline(self):
     def test_list_subscriptions(self):
         TOPIC_NAME = 'list-sub' + unique_resource_id('-')
         topic = Config.CLIENT.topic(TOPIC_NAME)
-        self.assertFalse(retry_unavailable(topic.exists)())
```

Comment:
I'm not sure what happens upstream but `topic.exists()` makes another API call to the service. Maybe the topic hasn't been indexed upstream yet in order for `topic.exists()` to work?

Seems like a question for the service devs?